templates/org/settings/options.tmpl

```diff
@@ -56,6 +56,16 @@
 							</div>
 						</div>
 
+						<div class="field" id="permission_box">
+							<label>{{.i18n.Tr "org.settings.permission"}}</label>
+							<div class="field">
+								<div class="ui checkbox">
+									<input class="hidden" type="checkbox" name="repo_admin_change_team_access" checked/>
```

Comment:
Always checked...  don't know how I could miss that.

Reply:
This was already merged. Better make another PR.